website/addons/github/static/githubFangornConfig.js

```diff
@@ -340,11 +337,27 @@ function _fangornColumns (item) {
     if(tb.options.placement === 'project-files') {
         columns.push(
         {
+            data  : 'size',
+            sortInclude : false,
+            filter : false,
+            custom : function() {return item.data.size ? $osf.humanFileSize(item.data.size, true) : '';}
+        });
+    }
+
+    if(tb.options.placement === 'project-files') {
```

Comment:
This looks like the same conditional as above; could you combine the `'downloads'` and `'size'` data to the same `.push` call?